ujt/callbacks/view_callbacks.py

```diff
@@ -101,38 +31,91 @@ def delete_view(delete_timestamps):
             },
             "value",
         ),
+        Input(
+            {
+                id_constants.CREATE_VIEW_BUTTON: ALL,
+            },
+            "n_clicks_timestamp",
+        ),
+        Input(
+            {
+                id_constants.DELETE_VIEW_BUTTON: id_constants.DELETE_VIEW_BUTTON,
+                "index": ALL,
+            },
+            "n_clicks_timestamp",
+        ),
     ],
-    prevent_initial_call=True,
 )
-def modify_view(tag_dropdown_values, style_dropdown_values):
-    """Updates the corresponding applied tag in the tag map.
+def update_view_store(
+    tag_dropdown_values, style_dropdown_values, create_timestamps, delete_timestamps
+) -> Tuple[List[Tuple[str, str]], str]:
+    """Saves the current list of views to the VIEW_STORE .
+
```

Comment:
This is the interesting change...
Dash's limitation of 1:1 mapping between callbacks and outputs is inconvenient, and is the reason why we need to combine all these cases into one callback.